lab3.md

```diff
@@ -80,8 +80,6 @@ functions:
     image: hello-openfaas
 ```
 
-On the line `image: ` make sure the name is prefixed with your Docker Hub account. For Alex Ellis this is `image: alexellis2/hello-openfaas`. Update this for your name every time you create a new function.
```

Comment:
Can we re-introduce a sentence like this and explain the `--prefix` flag at the same time?